python/event_generator.cpp

```diff
@@ -0,0 +1,137 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/pytypes.h>
+#include <pybind11/stl.h>
+
+#include <arbor/common_types.hpp>
+#include <arbor/event_generator.hpp>
+#include <arbor/schedule.hpp>
+
+#include "event_generator.hpp"
+#include "strings.hpp"
+
+namespace pyarb {
+
+template <typename Sched>
+event_generator_shim make_event_generator(
+        arb::cell_member_type target,
+        double weight,
+        const Sched& sched)
+{
+    return event_generator_shim(target, weight, sched.schedule());
+}
+
+void register_event_generators(pybind11::module& m) {
+    using namespace pybind11::literals;
+
+// Common schedules
+    // Regular schedule
+    pybind11::class_<regular_schedule_shim> regular_schedule(m, "regular_schedule",
+        "Describes a regular schedule with multiples of dt within the interval [tstart, tstop).");
+
+    regular_schedule
+        .def(pybind11::init<>(),
+            "Construct a regular schedule with default arguments:\n"
+            "  tstart: maximal numerical number.\n"
+            "  dt:     0 ms.\n"
+            "  tstop:  tstart.")
+        .def(pybind11::init<arb::time_type>(),
+            "dt"_a,
+            "Construct a regular schedule starting at tstart=0 ms, ending at tstop=maximal numerical number (in ms) with argument:\n"
+            "  dt:      The distance between time points in the regular sequence (in ms).\n")
+        .def(pybind11::init<arb::time_type, arb::time_type, arb::time_type>(),
+            "tstart"_a, "tstop"_a, "dt"_a,
+            "Construct a regular schedule with arguments:\n"
+            "  tstart: Events in the regular sequence start being delivered from this time (in ms).\n"
```

Comment:
"delivery time of first event in sequence (ms)"

Reply:
Done (added some "the" s).